static-site/src/components/splash/index.jsx

```diff
@@ -110,6 +110,13 @@ const Splash = () => {
           buttonText="Find out more"
           buttonLink="https://docs.nextstrain.org/en/latest/guides/communicate/narratives-intro.html"
         />
+        <Section
```

Comment:
These sections look better if the title is reduced to 32px and the padding above/below the abstract reduced.

(This is similar to a comment in the parent PR, and fine to leave out of this PR - I am happy to take a pass at styling post merge.)

Reply:
Continuing with #916